CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 2.0.2 (2020-06-19)
+
+  * Added a new timeout optional parameter to the v5 client to be able to pass
+    it the the underlying requests object. Without setting it the request can
+    hang forever if the server does not respond in a timely fashion. In case of
+    a timeout the request will not return a LuminosoTimeoutError.
```

Comment:
...where this says "will not return", is it supposed to say "will return"?

Reply:
Yes, typo - supposed to say "will now return" - will fix it